### PR TITLE
Tag box tag

### PR DESCRIPTION
**Pull request by shaobinspire**

### CODE REVIEW

File: Applications/Spire/Include/Spire/Ui/Tag.hpp [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+       * @param parent The parent widget.
+       */
+      explicit Tag(QString label, QWidget* parent = nullptr);
+
+      /** Returns <code>true</code> iff the Tag is deletable by the user. */
+      bool is_read_only() const;
+
+      /**
+       * Sets the read-only state.
+       * @param is_read_only True iff the Tag should be deletable.
```
Comment (spireken) [inline Applications/Spire/Include/Spire/Ui/Tag.hpp:29]: I think this should be: "should not be deletable."

**Review by spireken — CHANGES_REQUESTED** [review]
Looks good.

File: Applications/Spire/Include/Spire/Ui/Tag.hpp (hunk 1 above)
Comment (shaobinspire) [reply Applications/Spire/Include/Spire/Ui/Tag.hpp:29]: Good catch.

**Review by spireken — APPROVED** [review]

**Review by kranar — APPROVED** [review]
Flawless submission.

**Pull request merged**